libs/angular/src/lib/in-page-wizard-step-card/in-page-wizard-step-card.spec.ts

```diff
@@ -0,0 +1,222 @@
+import { Component } from '@angular/core'
+import {
+  render,
+  fireEvent,
+  RenderResult,
+  waitFor,
+} from '@testing-library/angular'
+import { NggInPageWizardStepCardComponent } from './in-page-wizard-step-card.component'
+
+describe('InPageWizardStepCardComponent', () => {
+  let component: NggInPageWizardStepCardComponent
+  let fakeComponent: RenderResult<NggFakeInPageWizardStepCardComponent>
+
+  beforeEach(async () => {
+    fakeComponent = await render(NggFakeInPageWizardStepCardComponent, {
+      declarations: [
+        NggFakeInPageWizardStepCardComponent,
+        NggInPageWizardStepCardComponent,
+      ],
+      providers: [],
+    })
+    fakeComponent.detectChanges()
+    component = fakeComponent.fixture.debugElement.children[0].componentInstance
+  })
+
+  it('should create', () => {
+    const rootElement = component
+    expect(rootElement).toBeDefined()
+  })
+  describe('Text input', () => {
+    it('should shown text inputs', async () => {
+      component.title = 'Test Title'
+      component.stepText = 'Test Step Text'
+
+      const title = await fakeComponent.findByTestId(
+        'in-page-wizard-step-card-title'
+      )
+      expect(title).toBeDefined()
+      expect(title.textContent).toEqual('Test Title')
+
+      const stepText = await fakeComponent.findByTestId(
+        'in-page-wizard-step-card-step-text'
+      )
+      expect(stepText).toBeDefined()
+      expect(stepText.textContent?.trim()).toEqual('Test Step Text')
```

Comment:
Noticed a weird bug that I suspect is connected to prettier setup. 
The prettier is formatting the HTML in a way that it adds a white-space before and after text that is directly under a div element. Did not see the same thing when using vs-code command "Save without formatting" and having the HTML in one line.

https://github.com/sebgroup/green/pull/853/files#diff-0f05445b29f895bce4dd447966508fea7a3ec664b8e9f6d84764572efe2b2826R20